com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ProcessConflictPreventer.java

```diff
@@ -66,13 +67,15 @@ private void validateNoConflictingOperationsExist(String mtaId, String spaceId)
                                .list();
     }
 
-    public void releaseLock(String processInstanceId) {
```

Comment:
`endedAt` can be hardcoded to `ZonedDateTime.now`. I don't foresee anyone wanting to pass some different value.

Reply:
If someone decide to use releaseLock method for different purpose and don't want to set final state, he can pass null and you won't have final state but you will have endedAt field already set.